CODE_OF_CONDUCT.md

```diff
@@ -3,7 +3,7 @@
 ## Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
+contributors and maintainers pledge to make participation in our project and
```

Comment:
`CODE_OF_CONDUCT.md` is untouched from Github. Also available here [www.kernel.org](https://www.kernel.org/doc/html/latest/process/code-of-conduct.html). I prefer leaving this file untouched.

Reply:
Alright - Might make a PR directly to them, then. I've changed it back.